models/cart.js

```diff
@@ -6,10 +6,13 @@ const cartSchema = new mongoose.Schema({
     ref: 'store',
     required: true
   },
-  contents: {
-    type: Array,
-    required: true
-  },
+  contents: [
+    {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'store',
```

Comment:
@tolls-3 Should this reference store or product?

Reply:
typo ... fixed it to reference product